Source/Core/VideoBackends/OGL/FramebufferManager.cpp

```diff
@@ -25,6 +24,144 @@
 #include "VideoCommon/VertexShaderGen.h"
 #include "VideoCommon/VideoBackendBase.h"
 
+constexpr const char* GLSL_REINTERPRET_PIXELFMT_SINGLELAYER = R"GLSL(
+void main(void) {
+  vec2 rawpos = vec2(gl_VertexID & 1, gl_VertexID & 2);
+  gl_Position = vec4(rawpos* 2.0 - 1.0, 0.0, 1.0);
+})GLSL";
+
+constexpr const char* GLSL_REINTERPRET_PIXELFMT_MULTILAYER = R"GLSL(
```

Comment:
Might be worth adding "VS" to the identifier to make the fact that it's a vertex shader clearer. Ditto for the fragment shaders.